rpcd/playbooks/ansible.cfg

```diff
@@ -12,19 +12,19 @@ timeout = 120
 
 # Set the path to the folder in os-ansible-deployment which holds the dynamic
 # inventory script - new config setting for ansible v1.9 and above
-inventory = /opt/os-ansible-deployment/playbooks/inventory/
+inventory = ../../os-ansible-deployment/playbooks/inventory/
```

Comment:
It can as an absolute path, as I tried to say.  Not seeing any other good simple solutions coming along so going to just do it that way.